city_scrapers/spiders/chi_southwest_home_equity_i.py

```diff
@@ -0,0 +1,173 @@
+import re
+from datetime import datetime, timedelta
+from io import BytesIO
+
+import requests
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text
+
+MONTH_REGEX = (
+    r"(?:Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)"
+    r"?|May|June?|July?|Aug(?:ust)?|Sep(?:tember)"
+    r"?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?)"
+)
+
+TIME_REGEX = r"(?:[1-9]|1[0-2]):[0-9]{2}\s(?:AM|PM)"
+
+
+class ChiSouthwestHomeEquityISpider(CityScrapersSpider):
+    name = "chi_southwest_home_equity_i"
+    agency = "Chicago Southwest Home Equity Commission I"
+    timezone = "America/Chicago"
+    allowed_domains = ["swhomeequity.com"]
+    start_urls = ["https://swhomeequity.com/agenda-%26-minutes"]
+    location = {
+        "name": "Southwest Home Equity Assurance office",
+        "address": "5334 W. 65th Street in Chicago, Illinois",
+    }
+
+    def parse(self, response):
+        table = response.xpath(
+            "body/div/div/div/div[2]/div[3]/div/div/section/div/div[2]"
+        )
+
+        for agenda_node in table.xpath(
```

Comment:
I think this approach makes sense, but their formatting doesn't seem very consistent, so we might have better luck using `defaultdict` to create a mapping of dates to links and then iterating over that to make sure we get everything. Here's an example in `chi_lsc_advisory` https://github.com/City-Bureau/city-scrapers/blob/a144119dd6c85a34ef7f18ffc01aa910274f2cd6/city_scrapers/spiders/chi_lsc_advisory.py#L37-L47